staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1122,6 +1124,14 @@ message ContainerStatusResponse {
     map<string, string> info = 2;
 }
 
+// ContainerResources holds resource limits configuration for a container.
+message ContainerResources {
+    // Resource limits configuration specific to Linux container.
+    LinuxContainerResources linux = 1;
+    // Resource limits configuration specific to Windows container.
+    WindowsContainerResources windows = 2;
+}
```

Comment:
Can you help me understand why this new struct is necessary? We didn't need this before to differentiate between Linux and Windows resources.

Reply:
Ah, this is explained in https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2273-kubelet-container-resources-cri-api-changes

/cc @marosset 
for windows CRI changes

Reply:
Also @claudiubelu @dcantah

Reply:
and @jsturtevant

Reply:
FWIW Windows (at least Windows Server 2022) and containerd have support for updating container resources https://github.com/containerd/containerd/pull/5778

I'm OK with not not having Windows support for this in Alpha but it'd be nice to have it before the feature goes stable.

Reply:
+@katiewasnothere

Reply:
@ehashman Looks like you found the answer.
@marosset Windows support is not planned for alpha. I would need someone working on Windows to help with this - hopefully before we go Beta. It's been a while since I checked with Patrick Lang about this.

Reply:
Unfortunately Patrick is no longer actively involved in the project. 
I can work with you in the future to help get folks with Windows expertise involved here.
Windows CRI changes look good to me!

Reply:
Sounds good. Let's plan to work on this around beta time.